brands/amenity/restaurant.json

```diff
@@ -1827,6 +1827,17 @@
       "cuisine": "pizza",
       "name": "Papa Gino's"
     }
+  },
+    "amenity/restaurant|PappaRich": {
+    "tags": {
+      "alt_name": "PappaRich Malaysian Restaurant",
```

Comment:
While there are no syntax errors, you might be better off removing this line altogether.

```suggestion
```

Remember that anything within `"tags": { }` will be added to new points at OpenStreetMap by default. Every time a mapper creates a node for PappaRich, or uses the validator to add the wikidata, they will also add that `alt_name`.

“Malaysian Restaurant” doesn't seem to be part of the name in any country, and `"cuisine": "malaysian"` and `"amenity": "restaurant"` already tag this as a Malaysian-style restaurant. Are you sure this name is worth keeping?

**Malaysian mappers have the last word on this, not us, not you.**

Reply:
Ok, I saw it at this tag
https://www.openstreetmap.org/node/6296999656
So mean that if I remote this alt name, then this xhain restaurant in Syndey also need to do so?

Reply:
It seems that there is only that single node in Australia that has the "alt_name" attached to it @MASangys, so I would actually remove it.

Reply:
Ok, just removed it. No problem